src/Npgsql/TypeMapping/GlobalTypeMapper.cs

```diff
@@ -88,6 +100,18 @@ public override bool UnmapEnum<TEnum>(string? pgName = null, INpgsqlNameTranslat
         }
     }
 
+    public override bool UnmapEnum(Type clrType, string? pgName = null, INpgsqlNameTranslator? nameTranslator = null)
```

Comment:
As in GlobalTypeMapper - have the generic version call the non-generic

Reply:
Done

Reply:
Done